packages/blade/src/components/Radio/__tests__/__snapshots__/Radio.native.test.tsx.snap

```diff
@@ -46,15 +46,16 @@ exports[`<Radio /> should render radio with label 1`] = `
       }
     >
       <Text
-        color="surface.text.subtle.lowContrast"
+        color="surface.text.subdued.lowContrast"
```

Comment:
The label thing is affecting radio as well. Are these intentional for Radio & Checkbox?

Reply:
yes there was a bug in the label color when the position is left